dnf5daemon-server/callbacks.cpp

```diff
@@ -81,7 +81,7 @@ int DownloadCB::progress(void * user_cb_data, double total_to_download, double d
 int DownloadCB::end(void * user_cb_data, TransferStatus status, const char * msg) {
     try {
         auto signal = create_signal_download(dnfdaemon::SIGNAL_DOWNLOAD_END, user_cb_data);
-        signal << static_cast<int>(status);
+        signal << static_cast<uint32_t>(status);
```

Comment:
This breaks API. Though unless the client performs more than 2^31, ABI will be unchanged. I will accept it into main because of no practical impact.

Reply:
Yeah, I wanted to unify data types used to transfer these "status"-like values. They are basically enums with only couple of available options so I do not expect to exceed maxint.